default/scripting/ship_hulls/organic/SH_ORGANIC.focs.txt

```diff
@@ -2,19 +2,17 @@ Hull
     name = "SH_ORGANIC"
     description = "SH_ORGANIC_DESC"
     speed = 90
-    fuel = 2
+    fuel = 6
     NoDefaultFuelEffect
     stealth = 5
-    structure = 5
+    structure = 3
     slots = [
-        Slot type = External position = (0.30, 0.25)
-        Slot type = External position = (0.50, 0.25)
-        Slot type = External position = (0.70, 0.25)
-        Slot type = Internal position = (0.60, 0.60)
+        Slot type = External position = (0.80, 0.50)
+        Slot type = Internal position = (0.60, 0.50)
     ]
-    buildcost = 14 * [[FLEET_UPKEEP_MULTIPLICATOR]] * [[SHIP_HULL_COST_MULTIPLIER]]
-    buildtime = 3
-    tags = [ "ORGANIC_HULL" "PEDIA_HULL_LINE_ORGANIC" "GOOD_FUEL_EFFICIENCY" ]
+    buildCost = 8 * [[FLEET_UPKEEP_MULTIPLICATOR]] * [[SHIP_HULL_COST_MULTIPLIER]]
```

Comment:
format was changed recently; these need to be lower-case letters

Reply:
done